docker/ubuntu/18.04/cpu_ispc_build/Dockerfile

```diff
@@ -29,8 +29,8 @@ RUN apt-get -y update && apt-get --no-install-recommends install -y software-pro
     apt-get --no-install-recommends install -y libstdc++-9-dev && \
     rm -rf /var/lib/apt/lists/*
 
-# Download and install required version of CMake. CMake 3.20 is required starting from LLVM 16.0.
-RUN if [[ $(uname -m) =~ "x86" ]]; then export CMAKE_URL="https://github.com/Kitware/CMake/releases/download/v3.20.5/cmake-3.20.5-linux-x86_64.sh"; else export CMAKE_URL="https://github.com/Kitware/CMake/releases/download/v3.20.5/cmake-3.20.5-linux-aarch64.sh"; fi && \
+# Download and install required version of cmake (3.23.5 for both x86 and aarch64) as required for superbuild preset jsons.
+RUN if [[ $(uname -m) =~ "x86" ]]; then export CMAKE_URL="https://cmake.org/files/v3.23/cmake-3.23.5-linux-x86_64.sh"; else export CMAKE_URL="https://github.com/Kitware/CMake/releases/download/v3.23.5/cmake-3.23.5-linux-aarch64.sh"; fi && \
     wget -q --retry-connrefused --waitretry=5 --read-timeout=20 --timeout=15 -t 5 $CMAKE_URL && mkdir /opt/cmake && sh cmake-*.sh --prefix=/opt/cmake --skip-license && \
```

Comment:
Should CMake installation be updated as it is in https://github.com/ispc/ispc/pull/2649/commits/17dda941f2c31a126ed1feb0c4aa5dee7eff08df?

Reply:
I have updated that, although this will probably conflict with the change in https://github.com/ispc/ispc/pull/2649. Whatever would be merged the first I will rebase the second.